lib/definitions/platform.d.ts

```diff
@@ -255,7 +255,7 @@ interface IPlatformData {
 	projectRoot: string;
 	normalizedPlatformName: string;
 	appDestinationDirectoryPath: string;
-	deviceBuildOutputPath: string;
+	deviceBuildOutputPath(options: IRelease): string;
```

Comment:
I believe we should rename this property to `getDeviceBuildOutputPath` as it is a better name for method.